.github/workflows/build.yml

```diff
@@ -97,6 +97,11 @@ jobs:
         run: |
           cat ./build/meson-logs/meson-log.txt
 
+      - name: Print meson test log
```

Comment:
This would be cosmetic change to how we handle failure, but maybe to avoid failing this step for example if build failed and we never run tests.

We could do separate `build` and `test` step and in `print` steps check
```
failure() && steps.build.outcome == 'failure'
```

Or something like that.

Reply:
Took me an embarrassingly long time to realize that github actions requires single quotes here... should be good now.